.github/workflows/ci-trigger-release.yml

```diff
@@ -0,0 +1,15 @@
+name: Trigger release
+
+on:
+  workflow_dispatch:
+
```

Comment:
Do we need to specify any workflow `permissions`? Or will this workflow inherit the permissions of the `github.actor`?